apis/k8ssandra/v1alpha1/k8ssandracluster_webhook.go

```diff
@@ -172,7 +173,9 @@ func (r *K8ssandraCluster) ValidateUpdate(old runtime.Object) (admission.Warning
 	if err := validateUpdateNumTokens(oldCluster.Spec.Cassandra, r.Spec.Cassandra); err != nil {
 		return nil, err
 	}
-
+	if DcRemoved(oldCluster.Spec, r.Spec) && DcAdded(oldCluster.Spec, r.Spec) {
```

Comment:
issue: This should be tested in the webhook_test.go also.

Reply:
Fair enough, done.